### ISSUE TITLE
DataTable.Driver: isDisplayingNothing: wrong name or fix it

### ISSUE THREAD

**Issue by hepiyellow** [wix/wix-style-react#1380]
DataTable.Driver: `isDisplayingNothing()` should either return `!element` (instead of `!!element`) or it should be renamed to `isDisplayingSomething()` or better yet `exists()`.

**Comment by hepiyellow** [comment]
It turns out that...

_Background facts_
- When DataTable's data is empty (and showHeaderWhenEmpty=false), it renders `null`.
- What ReactDOM is actually creating is Comment (DOM) Element. (node_type=8)
- A Comment Element has no `querySelectorAll` method.

_Current State_

- The driver's `isDisplayingNothing()` is returning `!!element`, which is ALWAYS true. Since when the component renders `null` the `element` is a Comment Element, not `null` or `undefined`.
- The spec test `should display nothing when data is empty` always passes.
- A spec test is missing for the case of non-empty data, that checks.
 `expect(driver.isDisplayingNothing()).toBe**Falsy**();`
- Other driver methods when data is empty, throw a `querySelectorAll is not a function`

_Suggested Solution_

- When driver is created do:
```
const isDisplayingNothing = element.nodeType===8; // 8=COMMENT_NODE
element = isDisplayingNothing? React.createElement("div") : element;
```
  - This will fix 
   - getCount() will return 0
   - All method which return an array, will return an empty aray.
   - All methods which need an row `index` will throw an out of bound index (instead of `querySelectorAll is not a function`)

Agree?

**Comment by argshook** [comment]
closed by #1395 (please reopen if i'm mistaken)
